doc/source/architecture.rst

```diff
@@ -44,21 +44,25 @@ The following diagram illustrates how PMM is currently structured:
 PMM Client
 ----------
 
-*PMM Client* is distributed as a tarball
-that you can install on any database host that you want to monitor.
-It consists of the following:
+*PMM Client* packages are available for most popular Linux distributions:
 
-* ``pmm-admin`` is a command-line tool for managing *PMM Client*
-  , for example, adding and removing database instances
+* DEB for Debian and Ubuntu
+* RPM for Red Hat Enterprise Linux and CentOS
```

Comment:
For Red Hat family of distributions, it also includes Amazon Linux, Oracle Linux etc.